src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -79,13 +83,16 @@ var ChartsPanel = React.createClass({
 
     onResize(size) {
         if (size) {
-            this.setState({ widthPx: (size.width-10), heightPx: (size.height-30), debouncedResize: this.state.debouncedResize });
+            this.state.debouncedResize.cancel;
+            this.setState({
+                widthPx: (size.width - 10),
+                heightPx: (size.height - 30),
+                debouncedResize: debounce(this.onResize, 200, {'leading':false, 'trailing':true, 'maxWait':10000})
+            });
```

Comment:
This is a more serious concern: Why do you have `debounceResize` in the state? It could be a method on the class object. Also, you are creating a new function with each setState which means that if `widthPx` and `heightPx` are the same the component will still re-render.